flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/relational/connection/JdbcConnectionPoolFactory.java

```diff
@@ -64,4 +65,6 @@ public HikariDataSource createPooledDataSource(JdbcSourceConfig sourceConfig) {
      * @return a database url.
      */
     public abstract String getJdbcUrl(JdbcSourceConfig sourceConfig);
+
+    public abstract String factoryIdentifier();
```

Comment:
If we use the class name as the identifier, we don't need to add this method which will always return a const value. What do you think?